constraints/miden-vm/bitwise.air

```diff
@@ -0,0 +1,70 @@
+def BitwiseAir
+
+# Enforces that column must be binary
+ev check_binary(main: [a]):
+    enf a^2 - a = 0
+
+# Returns value aggregated from limbs
+fn aggregate(limb: vector[4]) -> scalar:
+    return sum([2^i * a for (i, a) in (0..4, limb)])
+
+
+public_inputs:
+    stack_inputs: [16]
+
+trace_columns:
+    main: [s, a, b, a_limb[4], b_limb[4], zp, z, dummy]
+
+periodic_columns:
+    k0: [1, 0, 0, 0, 0, 0, 0, 0]
+    k1: [1, 1, 1, 1, 1, 1, 1, 0]
+
+boundary_constraints:
+    # This is a dummy trace column to satisfy requirement of at least one boundary constraint.
+    enf dummy.first = 0
+
+integrity_constraints:
+    # Enforce that selector must be binary
+    enf check_binary([s])
+
+    # Enforce that selector should stay the same throughout the cycle.
+    enf s' = s when k1
+
+    # Enforce that the input is decomposed into valid bits
+    enf check_binary([a]) for a in a_limb
+    enf check_binary([b]) for b in b_limb 
+
+    # Enforce that the value in the first row of column `a` of the current 8-row cycle should be 
+    # the aggregation of the decomposed bit columns `a_limb`.
+    enf a = aggregate([a_limb]) when k0
```

Comment:
minor nit: I think it should be `aggregate(a_limb)` for functions. Sorry, I missed this last time.